pandas/core/generic.py

```diff
@@ -5201,60 +5201,209 @@ def abs(self):
         """
         return np.abs(self)
 
```

Comment:
@jreback, if you refer to the existing "stable" ``pandas`` documentation, you can see that the klass setting appears to be the same for both ``Series`` and ``DataFrame``. 

First look at the "returns: value below the parameters [here](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.Series.describe.html):

![screenshot from 2016-12-28 07-51-02](https://cloud.githubusercontent.com/assets/9993/21525732/03debf3e-ccd4-11e6-9f9c-e2b807983bbd.png)

And then again [here](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.describe.html):

![screenshot from 2016-12-28 07-51-21](https://cloud.githubusercontent.com/assets/9993/21525735/0b7d6ce0-ccd4-11e6-9f26-a1ac86a3847a.png)